tests/cases.py

```diff
@@ -1508,8 +1512,7 @@ def test_model(self):
             embedding_dim=self.shape[1],
             entity_initializer=self.initializer,
             random_seed=0,
-            preferred_device="cpu",
-        )
+        ).to(resolve_device())
```

Comment:
why not just `.cpu()`?

Reply:
So you can run these tests on a GPU machine, and see whether they work there, too.